bin/tag.ml

```diff
@@ -59,9 +59,8 @@ let vcs_tag tag ~dry_run ~commit_ish ~force ~sign ~delete ~msg ~yes =
               commit_ish)
           ~default_answer:Yes)
       >>= fun () ->
-      let msg =
-        match msg with None -> strf "Distribution %s" tag | Some m -> m
```

Comment:
We should probably keep this line as well and add the changelog rather than completely replacing it.

Reply:
done